sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/SentryContentCachingRequestWrapper.java

```diff
@@ -0,0 +1,78 @@
+package io.sentry.spring.jakarta;
+
+import jakarta.servlet.ReadListener;
+import jakarta.servlet.ServletInputStream;
+import jakarta.servlet.http.HttpServletRequest;
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.springframework.web.util.ContentCachingRequestWrapper;
+
+public final class SentryContentCachingRequestWrapper extends ContentCachingRequestWrapper {
+  public SentryContentCachingRequestWrapper(HttpServletRequest request) {
+    super(request);
+  }
+
+  public SentryContentCachingRequestWrapper(HttpServletRequest request, int contentCacheLimit) {
+    super(request, contentCacheLimit);
+  }
+
+  @Override
+  public @NotNull ServletInputStream getInputStream() throws IOException {
```

Comment:
We're not quite sure, whether this change in behaviour can lead to different behaviour in either Spring or customers applications. It's probably safer to just create a util, that reads the cached content from `ContentCachingRequestWrapper.getContentAsByteArray` if the input stream is finished.